aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -606,13 +606,40 @@ Tensor& floor_divide_(Tensor& self, const Tensor& other) {
 // TODO: Make this structured to undo the perf regression from native:: removal
 // in call here
 Tensor mul(const Tensor& self, const Scalar& other) {
+  // prob ok if self never requires grad
```

Comment:
Not sure what this comment means? But no this should return a brand new Tensor.

Reply:
ahh yeah it's a stale comment. This is never called if `ZeroTensor` dispatch key is set.